### ISSUE TITLE
Mouse Press blocked by Texture below

### ISSUE THREAD

**Issue by Grosskopf** [godotengine/godot#44052]
**Godot version:**
3.2.3

**OS/device including version:**
Intel Processor on Opensuse Linux, GLES3

**Issue description:**
I had a button I wanted in front of my background Textures, so I parented it to a Node2D and increased the Node2D's Z value so i could see the Button. I was not able to Press the button until I set the mouse_filter of the Texture in the background to Ignore it.

**Steps to reproduce:**

create a scene with:
 - control
   - Node2D with Z value 1
     - Button
   - Texture Rect covering the Button's background, mouse_filter set to anything but ignore

see that you cannot press the button even though it is in front of the texture

**Minimal reproduction project:**
[Button_Press_Impossible.zip](https://github.com/godotengine/godot/files/5632387/Button_Press_Impossible.zip)

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot-proposals/issues/788.
